src/Command/Api4Command.php

```diff
@@ -149,11 +149,14 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       $columns = empty($params['select']) ? array_keys($result->first()) : $params['select'];
       $this->sendTable($input, $output, (array) $result, $columns);
     }
+    elseif (!empty($params['select']) && $params['select'] === ['row_count']) {
+      $this->sendResult($input, $output, $result->count());
+    }
```

Comment:
Aah, interesting, so you can `select` the `row_count` even though it's not actually a field in the result. This feels weird.... but OK, it appears to work w/AJAX. The AJAX-bindings wrap all results in an extra layer of JSON (`{values: ..., count: ...}`).

Of course, the patch above works if you select *only* the `row_count`. If you want to get data as well as `row_count`, then one needs to be more like the AJAX-bindings. I guess one could port the AJAX technique -- and for compatibility purposes, treat it as a flag, e.g.

```bash
cv api4 Contact.get +s display_name,row_count --wrap
```
```javascript
{
  "count": 123,
  "values": [
    {first},
    {second}
  ]
}
```

or like:

```bash
cv api4 Contact.get +s row_count --header
```
```javascript
{"count": 123}
[
  {first},
  {second}
]
```

FWIW, even with the wrapping, there's still a problem with chaining. You can see this in AJAX calls:

<img width="878" alt="Screen Shot 2021-05-14 at 6 07 57 PM" src="https://user-images.githubusercontent.com/1336047/118343719-5d3e4200-b4df-11eb-9e6d-ebd18e00067c.png">

I guess a couple questions:

* Looking forward -- Are there more header/wrapper fields that we should expect?
* Looking forward -- Do you expect some kind change/fix for the chaining issue?

If the answer is "No" on both fronts, then I'd be OK with this kind of special case. Otherwise, it's probably better to make the serialization formats more aligned.